tensorflow/python/keras/benchmarks/keras_examples_benchmarks/text_classification_transformer_benchmark_test.py

```diff
@@ -0,0 +1,239 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Benchmarks on Text classification with Transformer."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+import tensorflow as tf
+
+from tensorflow.python.keras.benchmarks import benchmark_util
+
+
+class TextWithTransformerBenchmark(tf.test.Benchmark):
+  """Benchmarks for Text classification with Transformer
+  using `tf.test.Benchmark`."""
+  def __init__(self):
+    super(TextWithTransformerBenchmark, self).__init__()
+    self.max_feature = 20000
+    self.max_len = 200
+    (self.imdb_x, self.imdb_y), _ = tf.keras.datasets.imdb.load_data(
+        num_words=self.max_feature)
+    self.imdb_x = tf.keras.preprocessing.sequence.pad_sequences(
+        self.imdb_x, maxlen=self.max_len)
+
+  def _build_model(self):
+    """model from https://keras.io/examples/nlp/text_classification_with_transformer/"""
+    embed_dim = 32
+    num_heads = 2
+    ff_dim = 32
+    inputs = tf.keras.layers.Input(shape=(self.max_len,))
+    embedding_layer = TokenAndPositionEmbedding(
+        self.max_len,
+        self.max_feature,
+        embed_dim)
+    x = embedding_layer(inputs) #pylint: disable=not-callable
```

Comment:
why pylint is here? is it necessary?

Reply:
If we don't add it, the `ubuntu_sanity` check will show the class not callable. But this class can be used. So I add this comments. Also, I ran locally and it works.

Reply:
SGTM.